### ISSUE TITLE
hello world tutorial error

### ISSUE THREAD

**Issue by finetjul** [InsightSoftwareConsortium/itk-wasm#781]
In the hello world tutorial, a folder named `wasi-build` is referenced. I personally (on Windows 11 , VSCode, git bash, itk-wasm@1.0.0-b.72) have only an `emscripten-build` folder, no `wasi-build` folder.

```
❯ npx itk-wasm run wasi-build/hello.wasi.wasm
Hello Wasm world!
```

It builds into the correct `wasi-build` folder with `itk-wasm@1.0.0-b.83`.

You might want to update the tutorial documentation accordingly.

**Comment by finetjul** [comment]
Other problems I had on Windows:
 - In loadEmscriptenModuleNode.js, I had to replace ```const fullModulePath = `${modulePrefix}.js`;``` with ```const fullModulePath = `file://${modulePrefix}.mjs`;```  and renaming `hello.js` into `hello.mjs` in `emscripten-build`.

First error:
```
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
```

Second error:
```
(node:25640) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
C:\Work\wasm\hello_world\emscripten-build\hello.js:3
  var _scriptDir = import.meta.url;
                          ^^^^

SyntaxError: Cannot use 'import.meta' outside a module
```

**Comment by finetjul** [comment]
Finally, here is the WASI error when trying to run the build:
```
PS C:\Work\wasm\hello_world> npx itk-wasm run wasi-build/hello.wasi.wasm
ERROR: Unknown option "--args,-e"

Usage: dockcross [options] [--] command [args]

By default, run the given *command* in an dockcross Docker container.

The *options* can be one of:

    --args|-a           Extra args to the *docker run* command
    --image|-i          Docker cross-compiler image to use
    --config|-c         Bash script to source before running this script


Additionally, there are special update commands:

    update-image
    update-script
    update

For update command help use: wasi-build/itk-wasm-build-env help <command>
undefined
```

**Comment by thewtex** [comment]
@finetjul thanks for the report!

For the first issue, here is a patch: https://github.com/InsightSoftwareConsortium/itk-wasm/pull/784

I will attempt to reproduce the others once on a Windows system.

**Comment by thewtex** [comment]
Addressed with #829 via #826 #798 #824
